UPGRADE.md

```diff
@@ -1,6 +1,8 @@
 # Upgrade
 
-## dev-master
+## dev-develop
+* Websocket start command changed to `app/console sulu:websocket:run`
+  * if you use xdebug on your server please start websocket with `app/console sulu:websocket:run -e prod`
```

Comment:
@dantleech already started to have a fluent text instead of only bullet points, I think we should agree on that, and be consistent here, even if it is just a short text.